src/controllers/articles.js

```diff
@@ -0,0 +1,96 @@
+import dotenv from 'dotenv';
+import { Article } from '../database/models';
+
+dotenv.config();
+
+export default {
+  create(req, res) {
+    const { title, description, content, image, slug } = req.body;
+    return Article.create({
+      ...req.body
+    })
+      .then(article =>
+        res.status(201).send({
```

Comment:
Please change this to the default responseGenerator: Please check here for more information